ddtrace/bootstrap/sitecustomize.py

```diff
@@ -85,12 +80,7 @@ def drop(module_name):
             _unloaded_modules.append(sys.modules[module_name])
         del sys.modules[module_name]
 
-    MODULES_REQUIRING_CLEANUP = ("gevent",)
-    do_cleanup = os.getenv("DD_UNLOAD_MODULES_FROM_SITECUSTOMIZE", default="auto").lower()
-    if do_cleanup == "auto":
-        do_cleanup = any(is_module_installed(m) for m in MODULES_REQUIRING_CLEANUP)
-
-    if not asbool(do_cleanup):
+    if not asbool(os.getenv("DD_UNLOAD_MODULES_FROM_SITECUSTOMIZE", default=True)):
```

Comment:
With this change we will always unload modules. There is no need to check if gevent is installed